packages/stable/indent-integration-okta/src/okta-api.ts

```diff
@@ -22,6 +19,9 @@ export async function callOktaAPI(
   status: Status
   response: BaseHttpResponse
 }> {
+  // Required for all authentication
+  const OKTA_DOMAIN = process.env.OKTA_DOMAIN || ''
```

Comment:
in order to prevent variables being set at first file read (and therefore before the loadSecrets call), I've needed to move them into the function calls themselves. Perhaps we should have a global `config.*` that we pull from instead of const variables, and then read from there?